src/admin-user-roles/admin-user-roles.routes.js

```diff
@@ -0,0 +1,118 @@
+ * version 3 of the License, or (at your option) any later version.
+ *  
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+ * See the GNU Affero General Public License for more details. You should have received a copy of
+ * the GNU Affero General Public License along with this program. If not, see
+ * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org. 
+ */
+
+(function() {
+
+    'use strict';
+
+    angular.module('admin-user-roles').config(routes);
+
+    routes.$inject = ['$stateProvider', 'ADMINISTRATION_RIGHTS', 'ROLE_TYPES'];
+
+    function routes($stateProvider, ADMINISTRATION_RIGHTS, ROLE_TYPES) {
+
+        $stateProvider.state('openlmis.administration.users.roles', {
+            abstract: true,
+            label: 'adminUserRoles.editUserRoles',
+            url: '/:id/roles',
+            accessRights: [ADMINISTRATION_RIGHTS.USERS_MANAGE],
+            resolve: {
+                roles: function(referencedataRoleFactory) {
+                    return referencedataRoleFactory.getAllWithType();
+                },
+                programs: function(programService) {
+                    return programService.getAll();
+                },
+                facilitiesMap: function(facilityService, ObjectMapper) {
+                    return facilityService.getAllMinimal()
+                        .then(function(facilities) {
+                            return new ObjectMapper().map(facilities);
+                        });
+                },
+                supervisoryNodes: function(AdminUserRolesSupervisoryNodeResource, facilitiesMap) {
+                    return new AdminUserRolesSupervisoryNodeResource(facilitiesMap).query()
+                        .then(function(page) {
+                            return page.content;
+                        });
+                },
+                warehouses: function(facilityService) {
+                    return facilityService.getAllMinimal();
+                },
+                user: function(userRoleAssignmentFactory, $stateParams, roles, programs, supervisoryNodes, warehouses) {
+                    return userRoleAssignmentFactory.getUser($stateParams.id, roles, programs, supervisoryNodes,
+                        warehouses);
+                },
+                roleRightsMap: function(roles, ObjectMapper) {
+                    return new ObjectMapper().map(roles, 'rights');
+                }
+            },
+            views: {
+                '@openlmis': {
+                    templateUrl: 'admin-user-roles/user-roles.html',
+                    controller: 'UserRolesController',
+                    controllerAs: 'vm'
+                }
+            }
+        });
+        // MALAWISUP-3888 Add checkboxes and column filters under roles when removing users roles: Starts here
```

Comment:
You can change it to // MALAWISUP-3888 Add checkboxes and column filters under roles when removing users roles **without starts here**